core/src/io/anuke/mindustry/core/Logic.java

```diff
@@ -82,7 +83,9 @@ public void play(){
             for(Team team : Team.all){
                 if(state.teams.isActive(team)){
                     for(Tile core : state.teams.get(team).cores){
-                        core.entity.items.add(Items.copper, 200);
+                        for(ItemStack itemStack : state.rules.loadout){
+                            core.entity.items.add(itemStack.item, itemStack.amount);
```

Comment:
I've been thinking about this code, and it may be a better idea to just set the starting items and clear everything else. Since core inventories are shared, it doesn't make sense to add a copy of the starter items for *per core*. Mapmakers would expect starter items to stay the same regardless of core count.

This would change the code to something like:
```java
if(!state.teams.get(team).cores.isEmpty()){
    TileEntity entity = state.teams.get(team).cores.first().entity;
    entity.items.clear();
    for(ItemStack stack : state.rules.loadout){
        entity.items.add(stack.item, stack.amount);
    }
}
```